.github/workflows/UnitTesting.yml

```diff
@@ -113,20 +118,22 @@ jobs:
       - name: ☑ Run unit tests
         if: matrix.system == 'windows'
         run: |
-          $PYTEST_ARGS = if ("${{ inputs.artifact }}".length -gt 0) { "--junitxml=TestReport.xml" } else { "" }
+          cd ${{ inputs.test_directory }}
+          $PYTEST_ARGS = if ("${{ inputs.artifact }}".length -gt 0) { "--junitxml=TestReportSummary.xml" } else { "" }
```

Comment:
Why change the name to "ReportSummary"? I find it either redundant or conflictive. It seems to imply a longer report exists?

On the other hand, might make this a top-level param.

Reply:
The jUnit file contains only a summary of all tests. The full test log is the e.g. the CI console log, right?

Reply:
Yes. But "Report" is already meaning that it's not the full log. So "ReportSummary" is saying the same twice.

Reply:
For me, a report is a document containing all the details.

Examples:
* A synthesis report contains all messages from synthesis.
* When you report at the military, you tell all details.
* When we write simulation reports for customers it contains all information we have about a testbench.

I know junit format supports embedded outputs from STDOUT etc. but it's not how it's widely used, right? That why I consider it a *report summary*.

Reply:
I suggest to split this modification, so we can focus on the dir modification and deal with this later.